rpc/conn.c

```diff
@@ -1448,11 +1449,15 @@ M0_INTERNAL int m0_rpc_conn_ha_timer_start(struct m0_rpc_conn *conn)
 
 M0_INTERNAL void m0_rpc_conn_ha_timer_stop(struct m0_rpc_conn *conn)
 {
+	M0_ENTRY("conn %p", conn);
 	M0_PRE(m0_rpc_machine_is_locked(conn->c_rpc_machine));
 	if (m0_sm_timer_is_armed(&conn->c_ha_timer)) {
 		M0_LOG(M0_DEBUG, "Cancelling HA timer; rpc conn=%p", conn);
 		m0_sm_timer_cancel(&conn->c_ha_timer);
 	}
+	else if (M0_IN(conn->c_ha_timer.tr_timer.t_state,
```

Comment:
[Style] Move else to the same line of above closed brace.

Reply:
Done